task/discover_granules_base.py

```diff
@@ -179,18 +179,20 @@ def create_file_mapping(self):
 
         return mapping
 
-    def lzards_output_generator(self, ret_dict):
+    def lzards_output_generator(self, granule_dict_list):
         """
         Generates a single dictionary generator that yields the expected cumulus output for a granule
-        :param ret_dict: Dictionary containing discovered granules, ETag, Last-Modified, and Size
+        :param granule_dict_list: Dictionary containing discovered granules, ETag, Last-Modified, and Size
```

Comment:
Can this description be updated to note that it's a list of dictionaries not a single dictionary?